src/plugins/visualizations/public/wizard/new_vis_modal.tsx

```diff
@@ -143,10 +144,12 @@ class NewVisModal extends React.Component<TypeSelectionProps, TypeSelectionState
     }
 
     let params;
-    if ('aliasPath' in visType) {
-      params = visType.aliasPath;
-      this.props.onClose();
-      this.navigate(visType.aliasApp, visType.aliasPath);
+    if ('alias' in visType) {
+      if (visType.alias !== INLINE_EDITING_ALIAS) {
+        params = visType.alias.path;
+        this.props.onClose();
+        this.navigate(visType.alias.app, visType.alias.path);
+      }
```

Comment:
Inline editing should **not** be allowed through the Visualization wizard - otherwise, you would end up with a flyout opening over top of a modal.